enacts/calc.py

```diff
@@ -746,6 +746,271 @@ def daily_to_dekad(daily_data, method=None, method_kwargs={}, time_dim="T"):
     return dekad_data
 
 
+def resample_interval_to_daily(time_series, extent=None, time_dim="T_bins"):
+    """ Resample any (interval-based) time series to daily
+
+    Parameters
+    ----------
+    time_series : xr.DataArray or xr.Dataset
+        data depending on time intervals greater or equal then a day
+    extent : str, optional
+        indicate the "extensive" or "intensive" property of `time_series` .
+        Upsampling to daily requires intensive data.
+        If extensive, make intensive by dividing by length of intervals in days
+        Default is None in which case: if units end with "/day", considers intensive,
```

Comment:
Why would we use this function if the input was already in units of */day?

Reply:
see extended Notes: e.g. monthly precipitation can be expressed in mm/day -- and you'd still want to resample that to daily